diff --git a/src/build/config/compiler/pgo/BUILD.gn b/src/build/config/compiler/pgo/BUILD.gn
--- a/src/build/config/compiler/pgo/BUILD.gn
+++ b/src/build/config/compiler/pgo/BUILD.gn
@@ -75,7 +75,7 @@ config("pgo_optimization_flags") {
       if (is_desktop_android && current_cpu == "x64") {
         _pgo_target = "android-desktop-x64"
       } else if (current_cpu == "arm64") {
-        _pgo_target = "android-arm64"
+        _pgo_target = "android-desktop-arm64"
       } else {
         _pgo_target = "android-arm32"
       }
@@ -106,6 +106,8 @@ config("pgo_optimization_flags") {
       inputs = [ "//chrome/build/android-arm32.pgo.txt" ]
     } else if (_pgo_target == "android-desktop-x64") {
       inputs = [ "//chrome/build/android-desktop-x64.pgo.txt" ]
+    } else if (_pgo_target == "android-desktop-arm64") {
+      inputs = [ "//chrome/build/android-desktop-arm64.pgo.txt" ]
     }
 
     if (_pgo_target != "" && pgo_data_path == "") {
