diff --git a/src/base/threading/platform_thread_android_stub.cc b/src/base/threading/platform_thread_android_stub.cc
index 9cce42e5..69f95509 100644
--- a/src/base/threading/platform_thread_android_stub.cc
+++ b/src/base/threading/platform_thread_android_stub.cc
@@ -1,4 +1,4 @@
-// Copyright (c) 2012 The Chromium Authors. All rights reserved.
+// Copyright 2012 The Chromium Authors
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
@@ -11,35 +11,33 @@
 #include <sys/types.h>
 #include <unistd.h>
 
+#include <optional>
+
 #include "base/lazy_instance.h"
 #include "base/logging.h"
 #include "base/threading/platform_thread_internal_posix.h"
 #include "base/threading/thread_id_name_manager.h"
-#include "third_party/abseil-cpp/absl/types/optional.h"
 
 namespace base {
 
 namespace internal {
 
 // - kRealtimeAudio corresponds to Android's PRIORITY_AUDIO = -16 value.
-// - kDisplay corresponds to Android's PRIORITY_DISPLAY = -4 value.
+// - kPresentation corresponds to Android's PRIORITY_DISPLAY = -4 value.
 // - kBackground corresponds to Android's PRIORITY_BACKGROUND = 10 value and can
 // result in heavy throttling and force the thread onto a little core on
 // big.LITTLE devices.
-const ThreadPriorityToNiceValuePairForTest
-    kThreadPriorityToNiceValueMapForTest[7] = {
-        {ThreadPriorityForTest::kRealtimeAudio, -16},
-        {ThreadPriorityForTest::kDisplay, -4},
-        {ThreadPriorityForTest::kNormal, 0},
-        {ThreadPriorityForTest::kUtility, 1},
-        {ThreadPriorityForTest::kBackground, 10},
+const ThreadTypeToNiceValuePairForTest kThreadTypeToNiceValueMapForTest[7] = {
+    {ThreadType::kRealtimeAudio, -16}, {ThreadType::kPresentation, -4},
+    {ThreadType::kDefault, 0},         {ThreadType::kUtility, 1},
+    {ThreadType::kBackground, 10},
 };
 
 // - kBackground corresponds to Android's PRIORITY_BACKGROUND = 10 value and can
 // result in heavy throttling and force the thread onto a little core on
 // big.LITTLE devices.
 // - kUtility corresponds to Android's THREAD_PRIORITY_LESS_FAVORABLE = 1 value.
-// - kDisplayCritical corresponds to Android's PRIORITY_DISPLAY = -4 value.
+// - kPresentation corresponds to Android's PRIORITY_DISPLAY = -4 value.
 // - kRealtimeAudio corresponds to Android's PRIORITY_AUDIO = -16 value.
 
 int ThreadTypeToNiceValue(const ThreadType thread_type) {
@@ -50,8 +48,8 @@ int ThreadTypeToNiceValue(const ThreadType thread_type) {
       return 1;
     case ThreadType::kDefault:
       return 0;
-    case ThreadType::kDisplayCritical:
-    case ThreadType::kInteractive:
+    case ThreadType::kPresentation:
+    case ThreadType::kAudioProcessing:
       return -4;
     case ThreadType::kRealtimeAudio:
       return -16;
@@ -62,14 +60,16 @@ bool CanSetThreadTypeToRealtimeAudio() {
   return false;
 }
 
+void SetCurrentThreadTypeImpl(ThreadType thread_type,
+                              MessagePumpType pump_type_hint) {}
+
 bool SetCurrentThreadTypeForPlatform(ThreadType thread_type,
                                      MessagePumpType pump_type_hint) {
   return false;
 }
 
-absl::optional<ThreadPriorityForTest>
-GetCurrentThreadPriorityForPlatformForTest() {
-  return absl::nullopt;
+std::optional<ThreadType> GetCurrentEffectiveThreadTypeForPlatformForTest() {
+  return std::nullopt;
 }
 
 }  // namespace internal
@@ -90,12 +90,9 @@ void PlatformThread::SetName(const std::string& name) {
     DPLOG(ERROR) << "prctl(PR_SET_NAME)";
 }
 
+void InitThreading() {}
 
-void InitThreading() {
-}
-
-void TerminateOnThread() {
-}
+void TerminateOnThread() {}
 
 size_t GetDefaultThreadStackSize(const pthread_attr_t& attributes) {
 #if !defined(ADDRESS_SANITIZER)
